src/coreclr/nativeaot/BuildIntegration/Microsoft.DotNet.ILCompiler.SingleEntry.targets

```diff
@@ -85,15 +85,10 @@
   </PropertyGroup>
 
   <!-- NativeAOT-LLVM: override ImportRuntimeIlcPackageTarget to not depend on the SDK-provided ResolvedILCompilerPack item -->
-  <Target Name="ImportRuntimeIlcPackageTarget" Condition="'$(BuildingFrameworkLibrary)' != 'true' and '$(_targetArchitecture)' == 'wasm'" DependsOnTargets="$(ImportRuntimeIlcPackageTargetDependsOn)" BeforeTargets="Publish">
-    <Error Condition="'@(PackageDefinitions)' == ''" Text="The PackageDefinitions ItemGroup is required for target ImportRuntimeIlcPackageTarget" />
-
-    <PropertyGroup>
-      <IlcHostPackagePath Condition="'%(PackageDefinitions.Name)' == '$(_hostPackageName)'">%(PackageDefinitions.ResolvedPath)</IlcHostPackagePath>
-      <RuntimePackagePath Condition="'%(PackageDefinitions.Name)' == '$(_targetPackageName)'">%(PackageDefinitions.ResolvedPath)</RuntimePackagePath>
-    </PropertyGroup>
-
-  </Target>
+  <PropertyGroup>
```

Comment:
```suggestion
  <!-- TODO-LLVM: <new issue number>. Remove once we upgrade to the .NET 10 SDK. -->
  <PropertyGroup>
```
Once the upstream integration settles in, `OverrideImportRuntimeIlcPackageTarget` will always be `true`.

Reply:
We can drop it entirely if everyone/everything migrates from packages to SDK integration